magento2/docker-compose.yml

```diff
@@ -0,0 +1,99 @@
+version: '3.3'
+
+services:
+  ################################################
+  ####          MariaDB Container          ######
+  ##############################################
+  db_m2:
+    image: nosto/mariadb:10.0
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: magento2_db
+
+  ################################################
+  ####          RabbitMQ Container         ######
+  ##############################################
+  rabbitmq_m2:
+    image: rabbitmq:3.7-management
```

Comment:
Why is this not https://hub.docker.com/layers/rabbitmq/library/rabbitmq/3.7/images/sha256-2fb8f0da539258741eae28031e533070660033c7b8698051c420be41b47b944b?context=explore

Reply:
Do we need the management tools?

Reply:
Easier to check how the queues are running/easier debugging

Reply:
👍